src/utils.js

```diff
@@ -0,0 +1,15 @@
+/* @flow */
+import type { LocaleType } from '@paypal/sdk-constants/src';
+
+const SPECIAL_COUNTRY_LANG_VARIANTS = {
+    'HK': 'zh_Hant'
```

Comment:
Added! and yes, this change seems to be a big deal, I just updated the PR, but let me try all together tomorrow, I'll share with you how I test all of this in order to validate it and make sure that I'm covering the needed cases properly.

Regarding to your previous comment, I'm not sure, specially because we are only having into account the special countries [here](https://github.com/paypal/paypal-sdk-client/pull/79/files#diff-978bff6f08d9f62278251257f4d62e5bf7e06dd48cf6db4bca16c37875a28e64R214) (`paypal-sdk-client`) .. however, there is a similar function [here](https://github.paypal.com/Checkout-R/clientsdknodeweb/blob/5eef78655ff88e21c1c90948c3cdac4d3c70f831/server/sdk/params.js#L156) in `clientsdknodeweb` . So, I need to duplicate the `getComputedLocales` function in both projects, but honestly, the scope of this change is a little blurry..  I'm going to do more tests and I'll let u know